### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lastranikos**
the example was improved to mount shared folder inside container from host

### CODE REVIEW

**Comment by tych0** [comment]
Hi, thanks for the pull.

It looks like the formatting is off here (probably you want the "or" not indented); can you also expand the language to "or, if your container already exists"?

Also, can you sign off your commit? (`git commit --amend -s`) Thanks!

**Comment by tych0** [comment]
It still looks a bit funny,

https://github.com/lastranikos/lxd/blob/512aec74a311f4fd808474edfb670f84d9992049/README.md#can-i-bind-mount-my-home-directory-in-a-container

**Comment by lastranikos** [comment]
Hi @tych0 and now?

**Comment by tych0** [comment]
Looks good, I think you still need the signed-off-by line, though.

**Comment by stgraber** [comment]
Oh wow, this branch is a mess :)

Looks like it's been signed-off but is in serious need of a rebase!

File: README.md [hunk 1]
```diff
@@ -311,8 +311,16 @@ lxc-devel, and we can escalate to CRIU lists as necessary.
...
 
-    lxc launch ubuntu priv -c security.privileged=true
-    lxc config device add priv homedir disk source=/home/$USER path=/home/ubuntu
+1.a) create a container.
+
+    lxc launch ubuntu privilegedContainerName -c security.privileged=true
+    
+1.b) or, if your container already exists.
+
+        lxc config set privilegedContainerName security.privileged true
```
Comment (stgraber) [inline README.md]: Should also include a "lxc restart privilegedContainerName" as converting from unprivileged to privileged requires a restart.

**Comment by stgraber** [comment]
Left one comment in-line. Can you fix this, then rebase your branch to just contain one commit that's properly signed-off?

Thanks!

**Pull request closed without merging**